Source/Core/Core/Boot/Boot.cpp

```diff
@@ -388,6 +388,11 @@ bool CBoot::BootUp()
       PowerPC::DBATUpdated();
       PowerPC::IBATUpdated();
 
+      // Because there is no TMD to get the requested system (IOS) version from,
+      // we default to IOS58, which is the version used by the Homebrew Channel.
+      if (dolLoader.IsWii())
+        SetupWiiMemory(0x000000010000003a);
```

Comment:
Doing it in this PR is fine IMO. But let's see if @booto has anything to say first.